docs/en/modules/admin/pages/pages/topics.adoc

```diff
@@ -1,11 +1,11 @@
 = Topics
 
-In this section, you can configure Topics, to organize your static pages and provide more clarity to the participants. 
+In this section, you can configure Topics, which will organize your static pages and provide more clarity to the participants.
 
 == Manage the topics
```

Comment:
```suggestion
== Configuration

=== Manage the topics
```

Reply:
@HHassig could you also make the "Create a new topic" heading a heading 3 to match the structure?